### PR TITLE
fix: -1 Days unsupported (International Date Line)

### PR DESCRIPTION
**Pull request by irma-kurnia-alaska**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #51 

## Summary:

- Adding unit test to testing when the -1 day arrive
- Adding condition to showing the span text in flight header when the --1 day arrive
- Adding demo for show the UI meanline -1 day arrive/depart

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: demo/demo.md [hunk 1]
```diff
@@ -98,11 +98,48 @@ This example illustrates a mainline nonstop with a next day arrival or departure
...
+<auro-accordion alignRight>
+  <span slot="trigger">See code</span>
+<!-- AURO-GENERATED-CONTENT:START (CODE:src=./../../apiExamples/nextDay.html) -->
+<!-- The below code snippet is automatically added from ./../../apiExamples/nextDay.html -->
+
+```html
+<auro-flight
+  flights='["AS 880"]'
+  duration="350"
+  departureTime="2022-05-31T22:55:00-10:00"
```
Comment (blackfalcon) [inline demo/demo.md:133]: There is no example file to support this inclusion. When a new build is ran, all of this new content is deleted. 

Please add all content to a file in `./apiExamples` and map reference in `./docs/partials/demo.md` file to build the `./demo/demo.md` file you edited.

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Please REBASE this branch from `main`, please DO NOT merge down.

File: demo/demo.md (hunk 1 above)
Comment (irma-kurnia-alaska) [reply demo/demo.md:133]: oh, sorry I will fix ASAP, thank you @blackfalcon.

**Review by blackfalcon — APPROVED** [review]

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**